### ISSUE TITLE
android client recorder faild!

### ISSUE THREAD

**Issue by foreverneverer** [Kurento/bugtracker#188]
**KMS Version**:

Version: 6.6.1
Found modules:
        Module: 'core' version '6.6.1'
        Module: 'elements' version '6.6.1'
        Module: 'filters' version '6.6.1'

**Other libraries versions**:

ii  gir1.2-gst-plugins-base-1.5            1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        Description: GObject introspection data for the GStreamer Plugins Base library
ii  gir1.2-gstreamer-1.5                   1.8.1.1~20160909144007.170.g0d6031b.trusty amd64        Description: GObject introspection data for the GStreamer library
ii  gstreamer1.5-alsa:amd64                1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer plugin for ALSA
ii  gstreamer1.5-libav:amd64               1.8.2.1~20160909143244.96.g493eee4.trusty  amd64        libav plugin for GStreamer
ii  gstreamer1.5-libav-dbg:amd64           1.8.2.1~20160909143244.96.g493eee4.trusty  amd64        libav plugin for GStreamer (debug symbols)
ii  gstreamer1.5-nice:amd64                0.1.13.1~20160930090742.81.geebfdab.trusty amd64        ICE library (GStreamer plugin)
ii  gstreamer1.5-plugins-bad:amd64         1.8.1.1~20160909144557.99.gf836e53.trusty  amd64        GStreamer plugins from the "bad" set
ii  gstreamer1.5-plugins-bad-dbg:amd64     1.8.1.1~20160909144557.99.gf836e53.trusty  amd64        GStreamer plugins from the "bad" set (debug symbols)
ii  gstreamer1.5-plugins-base:amd64        1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer plugins from the "base" set
ii  gstreamer1.5-plugins-base-dbg:amd64    1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer plugins from the "base" set
ii  gstreamer1.5-plugins-good:amd64        1.8.1.1~20160909143047.112.g9ee4248.trusty amd64        GStreamer plugins from the "good" set
ii  gstreamer1.5-plugins-good-dbg:amd64    1.8.1.1~20160909143047.112.g9ee4248.trusty amd64        GStreamer plugins from the "good" set
ii  gstreamer1.5-plugins-ugly:amd64        1.8.1.1~20160909192513.89.g2685b0f.trusty  amd64        GStreamer plugins from the "ugly" set
ii  gstreamer1.5-plugins-ugly-dbg:amd64    1.8.1.1~20160909192513.89.g2685b0f.trusty  amd64        GStreamer plugins from the "ugly" set (debug symbols)
ii  gstreamer1.5-pulseaudio:amd64          1.8.1.1~20160909143047.112.g9ee4248.trusty amd64        GStreamer plugin for PulseAudio
ii  gstreamer1.5-x:amd64                   1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer plugins for X11 and Pango
ii  kms-core-6.0                           6.6.1.20160930110533.trusty                amd64        Kurento core module
ii  kms-elements-6.0                       6.6.1.20160930122545.trusty                amd64        Kurento elements module
ii  kms-filters-6.0                        6.6.1.20160930131150.trusty                amd64        Kurento filters module
ii  kms-jsonrpc-1.0                        1.1.2.20160930094018.trusty                amd64        Kurento jsonrpc library
ii  kmsjsoncpp                             1.6.3.trusty.20170725122830.d78deb7        amd64        Kurento jsoncpp library
ii  kmsjsoncpp-dbg                         1.6.3.trusty.20170725122830.d78deb7        amd64        Kurento jsoncpp library
ii  kmsjsoncpp-dev                         1.6.3.trusty.20170725122830.d78deb7        amd64        Kurento jsonrpc library
ii  kurento-media-server-6.0               6.6.1.20160930143513.trusty                amd64        Kurento Media Server
ii  libgstreamer-plugins-bad1.5-0:amd64    1.8.1.1~20160909144557.99.gf836e53.trusty  amd64        GStreamer development files for libraries from the "bad" set
ii  libgstreamer-plugins-base1.5-0:amd64   1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer libraries from the "base" set
ii  libgstreamer-plugins-base1.5-dev       1.8.1.1~20160909142623.55.g7b19cfd.trusty  amd64        GStreamer development files for libraries from the "base" set
ii  libgstreamer1.5-0:amd64                1.8.1.1~20160909144007.170.g0d6031b.trusty amd64        Core GStreamer libraries and elements
ii  libgstreamer1.5-0-dbg:amd64            1.8.1.1~20160909144007.170.g0d6031b.trusty amd64        Core GStreamer libraries and elements
ii  libgstreamer1.5-dev                    1.8.1.1~20160909144007.170.g0d6031b.trusty amd64        GStreamer core development files
ii  libnice-dbg:amd64                      0.1.13.1~20160930090742.81.geebfdab.trusty amd64        ICE library (debugging symbols)
ii  libnice-dev                            0.1.13.1~20160930090742.81.geebfdab.trusty amd64        ICE library (development files)
ii  libnice10:amd64                        0.1.13.1~20160930090742.81.geebfdab.trusty amd64        ICE library (shared library)

**Client libraries**
- Language: java、JavaScript
- Version:6.6.0

**Browsers tested**
have tested this issue:
- Chrome:OK
- Firefox:OK
- Native:recorder only have audio，the video only show the first video clip

**System description:**
	KMS located in the public network，Client is android Client from “Nubotest”.

**What steps will reproduce the problem?**
When the KMS、kurento-room-demo start successfully,android Client publish the video.

**What is the expected result?**
android client recorded media is normal.
**What happens instead?**
the web can receive the video, but the recorded video only has audio,the video show the first video clip.
**Does it happen with one of the tutorials?**
kurento-room-demo
**Please provide any additional information below.**
recorde code:
> String pid = participantRequest.getParticipantId();
>     this.recorderCaller = new RecorderEndpoint.Builder(roomManager.getPipeline(pid),        RECORDING_PATH +  userName+ RECORDING_EXT ) .build();
>     this.passThru= new PassThrough.Builder(roomManager.getPipeline(pid)).build();
> 
>     roomManager.addMediaElement(pid, this.passThru);
>     this.passThru.connect(this.recorderCaller);
>     recorderCaller.record();

KMS log:
0:00:00.023408627  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/gst-plugins/faceoverlay/libfaceoverlay.so
0:00:00.023862679  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libfaceoverlay.so
0:00:00.024166912  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/gst-plugins/opencvfilter/libopencvfilter.so
0:00:00.028263692  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libopencvfilter.so
0:00:00.028433013  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/gst-plugins/movementdetector/libmovementdetector.so
0:00:00.065341206  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libmovementdetector.so
0:00:00.065805713  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/gst-plugins/facedetector/libfacedetector.so
0:00:00.068872379  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libfacedetector.so
0:00:00.069189471  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/gst-plugins/imageoverlay/libimageoverlay.so
0:00:00.071889333  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libimageoverlay.so
0:00:00.072161395  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/gst-plugins/logooverlay/liblogooverlay.so
0:00:00.074767355  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library liblogooverlay.so
0:00:00.075183340  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/server/libkmsfiltersimpl.so
0:00:00.078803368  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmsfiltersimpl.so
0:00:00.079351380  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-filters/src/server/libkmsfiltersmodule.so
0:00:00.083861926  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:135:loadModule: Loaded filters version 6.7.0~1.g0314843 generated at Sep  5 2017 09:50:13
0:00:00.086172687  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-jsonrpc/src/libjsonrpc.so
0:00:00.086384564  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libjsonrpc.so
0:00:00.088092633  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-core/src/gst-plugins/vp8parse/libvp8parse.so
0:00:00.088510188  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libvp8parse.so
0:00:00.088713033  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-core/src/gst-plugins/commons/rtpsync/libkmsrtpsync.so
0:00:00.088889588  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmsrtpsync.so
0:00:00.089414370  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-core/src/gst-plugins/commons/sdpagent/libkmssdpagent.so
0:00:00.089626229  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmssdpagent.so
0:00:00.089725838  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-core/src/gst-plugins/commons/libkmsgstcommons.so
0:00:00.090024160  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmsgstcommons.so
0:00:00.090659368  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-core/src/gst-plugins/libkmscoreplugins.so
0:00:00.091466150  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmscoreplugins.so
0:00:00.092266914  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-core/src/server/libkmscoremodule.so
0:00:00.094843076  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:135:loadModule: Loaded core version 6.7.0~1.g1a05fae generated at Sep  5 2017 09:46:28
0:00:00.094985607  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-core/src/server/libkmscoreimpl.so
0:00:00.095325983  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmscoreimpl.so
0:00:00.098465985  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/gst-plugins/rtpendpoint/librtpendpoint.so
0:00:00.102097205  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library librtpendpoint.so
0:00:00.102455343  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/gst-plugins/recorderendpoint/librecorderendpoint.so
0:00:00.103198381  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library librecorderendpoint.so
0:00:00.103642962  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/gst-plugins/rtcpdemux/librtcpdemux.so
0:00:00.103944892  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library librtcpdemux.so
0:00:00.104075810  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/gst-plugins/libkmselementsplugins.so
0:00:00.104849457  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmselementsplugins.so
0:00:00.105034196  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/gst-plugins/webrtcendpoint/libwebrtcendpoint.so
0:00:00.109364341  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libwebrtcendpoint.so
0:00:00.109913847  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/gst-plugins/webrtcendpoint/libwebrtcdataproto.so
0:00:00.110590110  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libwebrtcdataproto.so
0:00:00.110765504  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/gst-plugins/webrtcendpoint/libkmswebrtcendpointlib.so
0:00:00.114796924  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmswebrtcendpointlib.so
0:00:00.115471196  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/server/libkmselementsmodule.so
0:00:00.139599851  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:135:loadModule: Loaded elements version 6.7.0~2.ga478df5 generated at Sep  5 2017 09:49:04
0:00:00.140400251  1608      0x116f380 INFO    KurentoModuleManager ModuleManager.cpp:173:loadModules: Found module: ./kms-elements/src/server/libkmselementsimpl.so
0:00:00.140924146  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:69:loadModule: Symbol 'getFactoryRegistrar' not found in library libkmselementsimpl.so
0:00:00.142926941  1608      0x116f380 WARN    KurentoModuleManager ModuleManager.cpp:161:loadModules: Unable to load modules from:  /usr/local/lib/x86_64-linux-gnu/kurento/modules, it is not a directory
0:00:00.143097683  1608      0x116f380 INFO      KurentoMediaServer main.cpp:238:main: Kmsc version: 6.7.0~2.g428fcd9
0:00:00.143200527  1608      0x116f380 INFO      KurentoMediaServer main.cpp:239:main: Compiled at: Sep  5 2017 09:51:21
0:00:00.143278854  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:230:loadConfig: Reading configuration from: ../kurento-media-server/kurento.conf.json
0:00:00.144704633  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:167:loadModulesConfigFromDir: Looking for config files in ./modules_config
0:00:00.144837796  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:167:loadModulesConfigFromDir: Looking for config files in ./modules_config/kurento
0:00:00.145105872  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:192:loadModulesConfigFromDir: Loaded module config from: ./modules_config/kurento/BaseRtpEndpoint.conf.ini
0:00:00.145320455  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:192:loadModulesConfigFromDir: Loaded module config from: ./modules_config/kurento/WebRtcEndpoint.conf.ini
0:00:00.145622763  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:192:loadModulesConfigFromDir: Loaded module config from: ./modules_config/kurento/HttpEndpoint.conf.ini
0:00:00.145960207  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:192:loadModulesConfigFromDir: Loaded module config from: ./modules_config/kurento/MediaElement.conf.ini
0:00:00.146309642  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:192:loadModulesConfigFromDir: Loaded module config from: ./modules_config/kurento/UriEndpoint.conf.ini
0:00:00.148073940  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:192:loadModulesConfigFromDir: Loaded module config from: ./modules_config/kurento/SdpEndpoint.conf.json
0:00:00.148191020  1608      0x116f380 INFO       KurentoLoadConfig loadConfig.cpp:246:loadConfig: Configuration loaded successfully
0:00:00.148511681  1608      0x116f380 INFO    KurentoServerMethods ServerMethods.cpp:89:ServerMethods: Not enough resources exception will be raised when resources reach 0.800000 
0:00:00.149956145  1608      0x116f380 INFO    KurentoWebSocketTransport WebSocketTransport.cpp:209:WebSocketTransport: Secure websocket server not enabled
0:00:00.150495221  1608      0x116f380 INFO      KurentoMediaServer main.cpp:256:main: Mediaserver started

0:00:49.203556502  1608 0x7f7e10003380 INFO    KurentoServerMethods ServerMethods.cpp:794:ping: WebSocket Ping/Pong

0:01:00.488630292  1608 0x7f7e10003380 INFO    KurentoRecorderEndpointImpl RecorderEndpointImpl.cpp:83:RecorderEndpointImpl: Set WEBM profile
0:01:00.557299528  1608 0x7f7e180112d0 FIXME                default gstutils.c:3766:gst_pad_create_stream_id_internal:<videoSrc:src> Creating random stream-id, consider implementing a deterministic way of creating a stream-id
0:01:00.557579835  1608 0x7f7e18011280 FIXME                default gstutils.c:3766:gst_pad_create_stream_id_internal:<audioSrc:src> Creating random stream-id, consider implementing a deterministic way of creating a stream-id
0:01:00.579453055  1608      0x117d8a0 INFO    KurentoRecorderEndpointImpl RecorderEndpointImpl.cpp:83:RecorderEndpointImpl: Set WEBM profile
0:01:00.619303685  1608 0x7f7e1403cd40 FIXME                default gstutils.c:3766:gst_pad_create_stream_id_internal:<videoSrc:src> Creating random stream-id, consider implementing a deterministic way of creating a stream-id
0:01:00.619500700  1608 0x7f7e1403ccf0 FIXME                default gstutils.c:3766:gst_pad_create_stream_id_internal:<audioSrc:src> Creating random stream-id, consider implementing a deterministic way of creating a stream-id
0:01:04.491560877  1608      0x117d8a0 INFO    KurentoWebRtcEndpointImpl WebRtcEndpointImpl.cpp:151:generateDefaultCertificates: Unable to load the RSA certificate from file. Using the default certificate.
0:01:04.667632637  1608      0x117d8a0 INFO    KurentoWebRtcEndpointImpl WebRtcEndpointImpl.cpp:161:generateDefaultCertificates: Unable to load the ECDSA certificate from file. Using the default certificate.
0:01:04.668421942  1608      0x117d8a0 INFO    KurentoWebRtcEndpointImpl WebRtcEndpointImpl.cpp:96:remove_not_supported_codecs_from_array:<kmswebrtcendpoint0> Removing not supported codec 'AMR/8000'
0:01:04.668528182  1608      0x117d8a0 INFO    KurentoWebRtcEndpointImpl WebRtcEndpointImpl.cpp:96:remove_not_supported_codecs_from_array:<kmswebrtcendpoint0> Removing not supported codec 'H264/90000'
0:01:04.668684430  1608      0x117d8a0 INFO    KurentoWebRtcEndpointImpl WebRtcEndpointImpl.cpp:455:WebRtcEndpointImpl: STUN server Port not found in config; using default value: 3478
0:01:04.668806772  1608      0x117d8a0 INFO    KurentoWebRtcEndpointImpl WebRtcEndpointImpl.cpp:465:WebRtcEndpointImpl: STUN server IP address not found in config; NAT traversal requires either STUN or TURN server
0:01:04.668932643  1608      0x117d8a0 INFO    KurentoWebRtcEndpointImpl WebRtcEndpointImpl.cpp:483:WebRtcEndpointImpl: TURN server IP address not found in config; NAT traversal requires either STUN or TURN server
0:01:04.669509427  1608      0x117d8a0 INFO        kmswebrtcsession kmswebrtcsession.c:1514:kms_webrtc_session_parse_turn_url:<kmswebrtcsession0> TURN server info cleared
0:01:05.038595439  1608 0x7f7e1409a370 FIXME               basesink gstbasesink.c:3125:gst_base_sink_default_event:<nicesink0> stream-start event without group-id. Consider implementing group-id handling in the upstream elements
0:01:05.039682318  1608 0x7f7e10003380 WARN       kmsbasertpsession kmsbasertpsession.c:714:kms_base_rtp_session_start_transport_send:<kmswebrtcsession0> Cannot configure connection for media (id=2)
0:01:05.043484717  1608 0x7f7e140802d0 FIXME                default gstutils.c:3766:gst_pad_create_stream_id_internal:<nicesrc0:src> Creating random stream-id, consider implementing a deterministic way of creating a stream-id
0:01:06.081458587  1608 0x7f7e18063450 FIXME               basesink gstbasesink.c:3125:gst_base_sink_default_event:<nicesink0> stream-start event without group-id. Consider implementing group-id handling in the upstream elements
0:01:06.500555622  1608 0x7f7e180a46d0 INFO                kmsutils kmsutils.c:1426:kms_utils_adjust_output_pts:<rtpopusdepay0> Adding adjust PTS algorithm
0:01:06.512248422  1608 0x7f7e180a46d0 WARN         rtpsynchronizer kmsrtpsynchronizer.c:430:kms_rtp_synchronizer_process_rtp_buffer_mapped:<KmsRtpSynchronizer@0x7f7e14016ec0> Non monotonic PTS assignment in sorted mode (ssrc: 3412295877, seq: 17241, ts: 241034111, ext_ts: 241034111). Forcing monotonic
0:01:06.512390875  1608 0x7f7e180a46d0 WARN                kmsutils kmsutils.c:1384:kms_rtp_receiver_adjust_pts:<rtpopusdepay0> Non incremental PTS (last PTS: 0:00:06.112608948, PTS: 0:00:06.112608948, new PTS: 0:00:06.113608948)
0:01:06.512492387  1608 0x7f7e180a46d0 WARN         rtpsynchronizer kmsrtpsynchronizer.c:430:kms_rtp_synchronizer_process_rtp_buffer_mapped:<KmsRtpSynchronizer@0x7f7e14016ec0> Non monotonic PTS assignment in sorted mode (ssrc: 3412295877, seq: 17242, ts: 241035071, ext_ts: 241035071). Forcing monotonic
0:01:06.512609165  1608 0x7f7e180a46d0 WARN                kmsutils kmsutils.c:1384:kms_rtp_receiver_adjust_pts:<rtpopusdepay0> Non incremental PTS (last PTS: 0:00:06.113608948, PTS: 0:00:06.112608948, new PTS: 0:00:06.114608948)
0:01:06.527495368  1608 0x7f7e1801e2d0 FIXME               basesink gstbasesink.c:3125:gst_base_sink_default_event:<datasink_0> stream-start event without group-id. Consider implementing group-id handling in the upstream elements
0:01:06.528919433  1608 0x7f7e1801e370 FIXME                default gstutils.c:3766:gst_pad_create_stream_id_internal:<datasrc_0:src> Creating random stream-id, consider implementing a deterministic way of creating a stream-id
0:01:06.529294943  1608 0x7f7e0802bde0 FIXME                default gstutils.c:3766:gst_pad_create_stream_id_internal:<appsrc_stream_1:src> Creating random stream-id, consider implementing a deterministic way of creating a stream-id
0:01:06.589188656  1608 0x7f7e140802d0 WARN                 srtpdec gstsrtpdec.c:1149:gst_srtp_dec_decode_buffer:<srtp-decoder:rtp_sink> Unable to unprotect buffer (unprotect failed code 9)
0:01:06.589339018  1608 0x7f7e140802d0 WARN                 srtpdec gstsrtpdec.c:1178:gst_srtp_dec_decode_buffer:<srtp-decoder> Other error, dropping
0:01:06.833383285  1608 0x7f7e04006000 WARN         rtpsynchronizer kmsrtpsynchronizer.c:281:kms_rtp_synchronizer_rtp_diff_full:<KmsRtpSynchronizer@0x7f7e14016f60> Diff RTP ns time greater than base PTS, setting to 0
0:01:06.834743351  1608 0x7f7e04006000 INFO                kmsutils kmsutils.c:1426:kms_utils_adjust_output_pts:<rtpvp8depay0> Adding adjust PTS algorithm
0:01:06.919911079  1608 0x7f7e1409a050 WARN                 kmsremb kmsremb.c:306:kms_remb_local_update:<RTPSession@0x7f7e14094560> No probe, and bitrate == 0
0:01:07.288600519  1608 0x7f7e1409a050 INFO                 kmsremb kmsremb.c:467:kms_remb_local_on_sending_rtcp:<RTPSession@0x7f7e14094560> REMB: Send local bitrate estimation: 500000
0:01:07.778467089  1608 0x7f7e04006000 WARN         rtpsynchronizer kmsrtpsynchronizer.c:281:kms_rtp_synchronizer_rtp_diff_full:<KmsRtpSynchronizer@0x7f7e14016f60> Diff RTP ns time greater than base PTS, setting to 0
0:01:07.832447486  1608 0x7f7e04006000 WARN                kmsutils kmsutils.c:1384:kms_rtp_receiver_adjust_pts:<rtpvp8depay0> Non incremental PTS (last PTS: 0:00:00.000000000, PTS: 0:00:00.000000000, new PTS: 0:00:00.001000000)
0:01:08.005745356  1608 0x7f7e04006000 WARN         rtpsynchronizer kmsrtpsynchronizer.c:281:kms_rtp_synchronizer_rtp_diff_full:<KmsRtpSynchronizer@0x7f7e14016f60> Diff RTP ns time greater than base PTS, setting to 0
0:01:08.030499099  1608 0x7f7e04006000 WARN                kmsutils kmsutils.c:1384:kms_rtp_receiver_adjust_pts:<rtpvp8depay0> Non incremental PTS (last PTS: 0:00:00.001000000, PTS: 0:00:00.000000000, new PTS: 0:00:00.002000000)
0:01:08.036643441  1608 0x7f7e180112d0 FIXME               basesink gstbasesink.c:3125:gst_base_sink_default_event:<filesink0> stream-start event without group-id. Consider implementing group-id handling in the upstream elements
0:01:08.043369059  1608 0x7f7e1403cd40 FIXME               basesink gstbasesink.c:3125:gst_base_sink_default_event:<filesink1> stream-start event without group-id. Consider implementing group-id handling in the upstream elements
0:01:08.268421118  1608 0x7f7e04006000 WARN         rtpsynchronizer kmsrtpsynchronizer.c:281:kms_rtp_synchronizer_rtp_diff_full:<KmsRtpSynchronizer@0x7f7e14016f60> Diff RTP ns time greater than base PTS, setting to 0
0:01:08.350487460  1608 0x7f7e04006000 WARN                kmsutils kmsutils.c:1384:kms_rtp_receiver_adjust_pts:<rtpvp8depay0> Non incremental PTS (last PTS: 0:00:00.002000000, PTS: 0:00:00.000000000, new PTS: 0:00:00.003000000)
0:01:08.402733511  1608 0x7f7e04006000 WARN         rtpsynchronizer kmsrtpsynchronizer.c:281:kms_rtp_synchronizer_rtp_diff_full:<KmsRtpSynchronizer@0x7f7e14016f60> Diff RTP ns time greater than base PTS, setting to 0
0:01:08.451583136  1608 0x7f7e04006000 WARN                kmsutils kmsutils.c:1384:kms_rtp_receiver_adjust_pts:<rtpvp8depay0> Non incremental PTS (last PTS: 0:00:00.003000000, PTS: 0:00:00.000000000, new PTS: 0:00:00.004000000)
0:01:08.630399747  1608 0x7f7e04006000 WARN         rtpsynchronizer kmsrtpsynchronizer.c:281:kms_rtp_synchronizer_rtp_diff_full:<KmsRtpSynchronizer@0x7f7e14016f60> Diff RTP ns time greater than base PTS, setting to 0
0:01:08.662467269  1608 0x7f7e04006000 WARN                kmsutils kmsutils.c:1384:kms_rtp_receiver_adjust_pts:<rtpvp8depay0> Non incremental PTS (last PTS: 0:00:00.004000000, PTS: 0:00:00.000000000, new PTS: 0:00:00.005000000)

**Comment by stonyz** [comment]
I have the same issue, but not in android, it's in iOS (the video from android replay well).  after I checked the bad recorded file, I found the DST of video start from a very big integer instead of 0. That's the cause, but I don't know why the issue appear. and only on iOS. 
Just share my finding to you.

**Comment by eyison** [comment]
I also have  similar problem, function looks like normal, but sometimes below logs is huge, what's wrong , how to resolve
 
```
2020-07-15T16:52:42,667035 12998 0x00007fa4997fa700 warning rtpsynchronizer           kmsrtpsynchronizer.c:344 kms_rtp_synchronizer_rtp_diff_full() <KmsRtpSynchronizer@0x7fa4b80c3d60>  Diff RTP ns time greater than base PTS, setting to 0
2020-07-15T16:52:42,667114 12998 0x00007fa4997fa700 warning kmsutils                  kmsutils.c:1428 kms_utils_depayloader_adjust_pts_out() <rtpvp8depay77>  Fix PTS not strictly increasing, last: 0:00:00.604000000, current: 0:00:00.000000000, fixed = last + 1: 0:00:00.605000000
2020-07-15T16:52:42,746705 12998 0x00007fa4997fa700 warning rtpsynchronizer           kmsrtpsynchronizer.c:344 kms_rtp_synchronizer_rtp_diff_full() <KmsRtpSynchronizer@0x7fa4b80c3d60>  Diff RTP ns time greater than base PTS, setting to 0
2020-07-15T16:52:42,748079 12998 0x00007fa4997fa700 warning kmsutils                  kmsutils.c:1428 kms_utils_depayloader_adjust_pts_out() <rtpvp8depay77>  Fix PTS not strictly increasing, last: 0:00:00.605000000, current: 0:00:00.000000000, fixed = last + 1: 0:00:00.606000000
```

**Comment by j1elo** [comment]
I think this should have been fixed by https://github.com/Kurento/kms-core/pull/29, https://github.com/Kurento/kms-core/commit/d132cb7c2d1bfcb00c12829367c144e058f35815
